lib/carbon/conf.py

```diff
@@ -504,6 +504,10 @@ def get_default_parser(usage="%prog [options] <start|stop|status>"):
         "--instance",
         default='a',
         help="Manage a specific carbon instance")
+    parser.add_option(
+        "--logger",
+        default='None',
+        help="Use a log observer factory for the initial log observer")
```

Comment:
Can we use the same text as in the PR description here?  Also does it make sense to add support for `--syslog` and `--logfile` as well for completeness?

Reply:
`--syslog` is already supported https://github.com/graphite-project/carbon/blob/master/lib/carbon/conf.py#L470

Added support for `--logfile` as well